conda/environment.devenv.yml

```diff
@@ -60,9 +60,8 @@ dependencies:
 - cmake
 - coin3d
 - compilers
-- conda-build
```

Comment:
why remove conda-build? in any case we could replace it with boa

Reply:
yes they aren't strictly necesary, but are still worth including IMO. Are there any problems with these packages?

Reply:
I've restored the files, if desired a separate PR will be made to remove them.